platform/functions/react-server/regional-server.mjs

```diff
@@ -127,6 +127,21 @@ const createApigHandler = (build) => {
   });
 };
 
+const accumulateCookies = (headers) => {
+  // node >= 19.7.0 with no remix fetch polyfill
+  if (typeof headers.getSetCookie === "function") {
+    return headers.getSetCookie();
+  }
+  // node < 19.7.0 or with remix fetch polyfill
+  const cookies = [];
+  for (let [key, value] of headers.entries()) {
+    if (key === "set-cookie") {
+      cookies.push(value);
```

Comment:
Should not this be in format

```
`${key}=${value}`
```

Based on this [MDN example](https://developer.mozilla.org/en-US/docs/Web/API/Headers/getSetCookie) it should be.

```js
headers.getSetCookie();
// Returns ["name1=value1", "name2=value2"]
```

Reply:
so yes, but in this case the key is not the name of the cookie, but rather the key of the header. the value will contain the whole cookie header